x-pack/test_serverless/functional/test_suites/security/config.example.context_awareness.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from '../../config.base';
+
+export default createTestConfig({
+  serverlessProject: 'security',
+  testFiles: [require.resolve('../common/discover/context_awareness')],
+  junit: {
+    reportName:
+      'Serverless Security Discover Context Awareness Functional Tests - Example Profiles',
+  },
+  kbnServerArgs: [
+    `--discover.experimental.enabledProfiles=${JSON.stringify([
+      'example-root-profile',
+      'example-solution-view-root-profile',
+      'example-data-source-profile',
+      'example-document-profile',
+    ])}`,
+  ],
+  // include settings from project controller
+  // https://github.com/elastic/project-controller/blob/main/internal/project/observability/config/elasticsearch.yml
+  esServerArgs: ['xpack.ml.dfa.enabled=false'],
```

Comment:
The [link](https://github.com/elastic/project-controller/blob/main/internal/project/observability/config/elasticsearch.yml) on line 26 leads to file that only contains comments, no values.  
Is this ok?